scripts/release/mule/sign/sign.sh

```diff
@@ -29,36 +30,53 @@ cd "$PKG_DIR"
 
 if [ -n "$S3_SOURCE" ]
 then
-    aws s3 sync "s3://$S3_SOURCE/$CHANNEL/$VERSION/$OS_TYPE/$ARCH_TYPE/" .
+    aws s3 cp --recursive --exclude "*" --include "*$CHANNEL*$VERSION*" "s3://$S3_SOURCE/$CHANNEL/$VERSION" .
 fi
 
 # TODO: "$PKG_TYPE" == "source"
 
-# Clean package directory of any previous operations.
-rm -rf hashes* *.sig *.asc *.asc.gz
-
-for file in *.tar.gz *.deb
-do
-    gpg -u "$SIGNING_KEY_ADDR" --detach-sign "$file"
-done
-
-for file in *.rpm
-do
-    gpg -u rpm@algorand.com --detach-sign "$file"
+# https://unix.stackexchange.com/a/46259
+# Grab the directories directly underneath (max-depth 1) ./tmp/node_pkgs/ into a space-delimited string.
+# This will help us target `linux`, `darwin` and (possibly) `windows` build assets.
+# Note the surrounding parens turns the string created by `find` into an array.
+OS_TYPES=($(find . -mindepth 1 -maxdepth 1 -type d -printf '%f\n'))
+for os in "${OS_TYPES[@]}"; do
+    if [ "$os" = linux ]
```

Comment:
Do we need to also sign the darwin builds?

Reply:
Yes, that's my understanding.

Reply:
Eventually, but we aren't building them yet. Because of the ARCHS loop below, it'd probably need to be a separate block anyway.